README.md

```diff
@@ -1,2 +1,55 @@
-# supervision-success-component
+# Supervision Success
+
 A React component powering a miniature web app for exploring the impact of success in supervision on the justice system at large
+
+## Contents
+
+1. [Input File Format](#input-file-format)
+
+## Input File Format
+
+Example of correct CSV could be found in [`params.csv`](/src/params.csv) file.
+
+First row of the table consists of next required params **in any order** (This params are used in application):
+
+- `state`
+- `newOffensePopulation`
+- `revocationA`
+- `revocationsTimescale`
+- `marginalCostPerInmate`
+- `totalCostPerInmate`
+- `checkpoint[n]`
+- `savings[n]`
+
+Checkpoint and according savings should be provided in such format:
+`checkpoint1`, `savings1` `checkpoint2`, `savings2` `...`, `checkpoint99`, `savings99`
+
+The number of checkpoints is not limited
+
+Any others params that may be useful for calculations also could be in the sheet in any format
+(This params are not used in application) e.g.:
+
+- `revocationsPopulation`
+- `Population Fraction`
+- `totalPopulation`
+- `revocationsAdmissions`
+- `admissionsFraction`
+- `Total Admissions`
+- `newOffenseAdmissions`
+- `newOffenseAvgTimeServedInYears`
+- `newOffenseA`
+- `...`
+
+Next rows should be populated with corresponding data. There should not be gaps between rows.
+
+Examples of correct params table:
```

Comment:
My worry about providing these carefully formatted tables is that some readers may think, "Oh, I should format it like this [Markdown] instead of as CSV." To help avoid that confusion, I would change this sentence to,

"Here are some additional examples of valid model parameters. Note: these are formatted as a table in Markdown for readability, but these tables are not valid CSV and the exact formatting should be provided as described above."